packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.11-rc5
+    tag: v1.0.11
   securityScan:
     repository: rancher/security-scan
-    tag: v0.2.10-rc5
+    tag: v0.2.11
```

Comment:
HOw come we are bumping from v0.2.10-rc5 to v0.2.11?

Reply:
v0.2.10-rc5 was created instead of v0.2.11-rc1 by mistake so i created v0.2.11 tag.
ref: https://github.com/rancher/security-scan/tags